server/routes/induction/update/previousWorkExperienceDetailUpdateController.ts

```diff
@@ -94,7 +94,7 @@ export default class PreviousWorkExperienceDetailUpdateController extends Previo
         return {
           ...experience,
         }
-      })
+      }) || []
```

Comment:
@nathanrussell-moj-digital - this is relevant to what you're currently working on.

Reply:
👍 Thanks